.cruft.json

```diff
@@ -0,0 +1,21 @@
+{
+  "template": "D:\\open-source\\gsoc\\cookiecutter-python",

```

Comment:
@uniqueg we need to merge and finalize the cookiecutter, when generated via gh, this shoul dbe gh link, that is not possible before merge.